db/migrate/20230109224336_remove_donations_from_site_banners.rb

```diff
@@ -0,0 +1,11 @@
+class RemoveDonationsFromSiteBanners < ActiveRecord::Migration[6.1]
+  def up
+    remove_column :site_banners, :default_donation
```

Comment:
I wonder if this will cause any protobuf type errors in the pCDN?

Reply:
I did edit the protobuf files to remove references to the `default_donations`.  I can run the pCDN job in staging before we push to master to make sure it won't cause more problems though.